centos/8/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:8
+MAINTAINER Alexey Kopytov <akopytov@gmail.com>
+
+# Fix missing locales
+ENV LANGUAGE="en_US.UTF-8"
```

Comment:
amd64 image uses `ENV LC_ALL="C" LANG="en_US.UTF-8"`. Is there any difference?

Reply:
Oops, I have no idea where this line came from. I just tested, setting `LANGUAGE` has no effect. I will make this line identical to the amd64 image.

Reply:
It seems that (almost) everything may be adjusted to be the same as for amd64 image. This way looks better for me, because it is easier in maintanence.

Reply:
OK, I made this file as identical as possible to the AMD64 version. Everything is identical now except the backports part.